dev-util/glslang/glslang-1.3.236.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..11} )
+inherit cmake-multilib python-any-r1
+
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/KhronosGroup/${PN}.git"
+	inherit git-r3
+else
```

Comment:
Have you done manual testing to confirm there's no ABI break? Unfortuantely we can't rely on anything but manual testing due to upstream being somewhat flaky.

Reply:
yeah if it is only for 9999 then that name is good

Reply:
but also pls check the abi thing, its very important

Reply:
@sarnex how can I check the 32-bit libraries of this package for sure?

Reply:
you dont need to check 32bit in particular, just try various vulkan apps that weren't recompiled

Reply:
well, 64bit version DOES work